xgboost_ray/tests/test_tune.py

```diff
@@ -65,14 +65,11 @@ def tearDown(self):
 
     # noinspection PyTypeChecker
     def testNumIters(self):
-        ray_params = RayParams(cpus_per_actor=1, num_actors=2)
+        ray_params = RayParams(cpus_per_actor=1, num_actors=1)
```

Comment:
Can we leave this at 2? This was actually an error before (the num iters was incorrect because we reported multiple times from different actor ranks). Keeping this will make sure we don't regress

Reply:
Oops that's my bad. I had changed this for some debugging and forgot to switch back. Thanks for pointing this out.